### ISSUE TITLE
Fix Vulnerabilities reported by ossindex

### ISSUE THREAD

**Issue by ckunki** [exasol/cloud-storage-extension#229]
* [org.apache.ivy:ivy:jar:2.5.0](https://ossindex.sonatype.org/component/pkg:maven/org.apache.ivy/ivy@2.5.0?utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1) in compile
    * CVE-2022-37865, severity CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') (9.1)
    * CVE-2022-37866, severity CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') (7.5)
  
  Excluded vulnerabilities:
    - 1 vulnerability found (6.2); https://ossindex.sonatype.org/vulnerability/sonatype-2020-0926
    - 1 vulnerability found (6.5); https://ossindex.sonatype.org/vulnerability/sonatype-2020-0026
    - [CVE-2019-10172] CWE-611: Improper Restriction of XML External Entity Reference ('XXE') (7.5); https://ossindex.sonatype.org/vulnerability/CVE-2019-10172?component-type=maven&component-name=org.codehaus.jackson%2Fjackson-mapper-asl&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
    - [CVE-2017-7525] CWE-184: Incomplete Blacklist (9.8); https://ossindex.sonatype.org/vulnerability/CVE-2017-7525?component-type=maven&component-name=org.codehaus.jackson%2Fjackson-mapper-asl&utm_source=ossindex-client&utm_medium=integration&utm_content=1.8.1
    - 1 vulnerability found (6.5); https://ossindex.sonatype.org/vulnerability/sonatype-2021-0818
    - 1 vulnerability found (8.2); https://ossindex.sonatype.org/vulnerability/sonatype-2022-5820
    - 1 vulnerability found (8.6); https://ossindex.sonatype.org/vulnerability/sonatype-2022-5732

**Comment by ckunki** [comment]
Available is [2.5.1](https://mvnrepository.com/artifact/org.apache.ivy/ivy/2.5.1)

**Comment by ckunki** [comment]
org.apache.ivy:ivy:jar:2.5.0 is a transitive dependency:
```
com.exasol:cloud-storage-extension:jar:2.6.1
\- org.apache.spark:spark-sql_2.13:jar:3.3.0:compile
   \- org.apache.spark:spark-core_2.13:jar:3.3.0:compile
      \- org.apache.ivy:ivy:jar:2.5.0:compile
```
